src/resources/js/components/Presentation.vue

```diff
@@ -47,20 +63,32 @@ export default {
     };
   },
   beforeMount() {
-    // set the bearer token
-    window.axios.defaults.headers.common = { Authorization: `Bearer ${localStorage.getItem('Authorization-token')}` };
-
-    // take the param in vu-route in presentation/{idPresentation}
-
-    const apiUrl = `/api/v1/presentations/${this.$route.params.idPresentation}`;
-
+    const apiUrl = `presentations/${this.$route.params.idPresentation}`;
     window.axios
       .get(apiUrl)
-      .then((response) => {
-        this.presentation = response.data;
+      .then((presResponse) => {
+        this.presentation = presResponse.data.data;
       })
-      .catch((error) => {
-        this.showMessage('error', `erreur lors de l'envoie des donnée ${error}`);
+      .catch(() => {
+        this.showMessage('error', 'Oops une erreur est survenue lors du traitement de votre demande');
+      });
+
+    // get current user
+    window.axios
+      .get('/me')
+      .then((userResponse) => {
+        this.user = userResponse.data.data;
+        // check if the user is subscribed
+        window.axios
+          .get(`/users/${this.user.id}/presentations`)
```

Comment:
didn't the linter have a hissy fit?!

https://github.com/airbnb/javascript#strings